### DISCUSSION TITLE
Static media does not get loaded

### DISCUSSION THREAD

**Discussion by germebl** [pretix/pretix#4721] [Q&A]
Hey folks,

i actually give pretix a try to check out how it performs on-premise. I followed the documentation step by step, completed it and be able to do anything, but the static data doesnt get loaded. First of all - i think it's cause i'm having the nginx reverse on another machine and that location does not work as expected when using that type of setup where the reverse is on another machine:

```
    location /static/ {
        alias /var/pretix/venv/lib/python3.11/site-packages/pretix/static.dist/;
        access_log off;
        expires 365d;
        add_header Cache-Control "public";
    }
```

So this is mostly a nginx misconfiguration, but maybe someone using something same and would share how i could solve it?

**Comment by raphaelm** [comment]
Yup that can't work then :) What we use in production is two nginx's ;)

**Reply by germebl** [reply]
Hey, thanks for the quick tip. Works fine!
